db/migrate/20230709065227_separate_role_access_restrictions_for_service_templates.rb

```diff
@@ -0,0 +1,32 @@
+class SeparateRoleAccessRestrictionsForServiceTemplates < ActiveRecord::Migration[6.0]
+  class MiqUserRole < ActiveRecord::Base; end
```

Comment:
Does it make sense to have settings as a serialized column here?

So we could read / write to it and we know we are not messing up some strange yaml thing?

Reply:
> Does it make sense to have settings as a serialized column here?

Thanks for the comment @kbrock, I think `serialize :settings` is a big improvement.